protobuf/grpc_service.proto

```diff
@@ -1035,6 +1035,60 @@ message InferBatchStatistics
   StatisticDuration compute_output = 4;
 }
 
+//@@
+//@@.. cpp:var:: message InferResponseStat
+//@@
+//@@   Inference response statistics.
+//@@
+message InferResponseStat 
+{
+  //@@  .. cpp:var:: StatisticDuration compute_infer
+  //@@
+  //@@     The count and cumulative duration that generate the inference
+  //@@     results for a response.
```

Comment:
Is this cumulative or for a single request/response? Can you share example statistics output?

Reply:
It is cumulative across multiple responses for a request.

Reply:
It is rewritten.